daffodil-runtime1/src/main/scala/org/apache/daffodil/infoset/XMLTextInfosetOutputter.scala

```diff
@@ -160,7 +163,13 @@ class XMLTextInfosetOutputter private (writer: java.io.Writer, pretty: Boolean)
         if (simple.erd.runtimeProperties.get(XMLTextInfoset.stringAsXml) == "true") {
           writeStringAsXml(simpleVal)
         } else {
-          writer.write(scala.xml.Utility.escape(remapped(simpleVal)))
+          val simpleValEscaped = xmlOutputStyle match {
+            case XMLOutputStyle.PrettyPrintSafe => {
+              "<![CDATA[%s]]>".format(remapped(simpleVal).replaceAll("]]>", "]]]]><![CDATA[>"))
+            }
+            case XMLOutputStyle.Standard => scala.xml.Utility.escape(remapped(simpleVal))
```

Comment:
I just now realize that this new enum is really just describing how we escape XML strings. `PrettyPrintSafe` is maybe the intention and a good reason why to use this, but I wonder if somethign like `InfosetStringEscapeStyle` with values `CDATA` and `Standard` might be more accurate? Or maybe staying generic allows us come up with other changes in the future that make things more safe for pretty printing?